.github/workflows/release-master.yaml

```diff
@@ -72,6 +72,24 @@ jobs:
           path: /tmp/digests/*
           if-no-files-found: error
           retention-days: 1
+  helm:
+    permissions:
+      contents: read
+      packages: write
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Install Helm
+        uses: azure/setup-helm@v3
+        with:
+          token: ${{ secrets.GH_PAT }}
```

Comment:
No lock on helm version?

Reply:
Yes we could do something like:
```YAML
- name: Install Helm
  uses: azure/setup-helm@v3
  with:
    version: '3.x'  # Specify the desired version
    token: ${{ secrets.GH_PAT }}
```

Which do you recommend?

Reply:
Recommend `3.12.3`
helm v3.13.0 contains bugs related to oci that will be fixed in v3.13.1. https://github.com/helm/helm/issues/12423